internal/winapi/cfgmgr.go

```diff
@@ -0,0 +1,6 @@
+package winapi
+
+//sys CMGetDeviceIDListSize(pulLen *uint32, pszFilter *byte, uFlags uint32) (hr error) = cfgmgr32.CM_Get_Device_ID_List_SizeA
+//sys CMGetDeviceIDList(pszFilter *byte, buffer *byte, bufferLen uint32, uFlags uint32) (hr error)= cfgmgr32.CM_Get_Device_ID_ListA
+//sys CMLocateDevNode(pdnDevInst *uint32, pDeviceID string, uFlags uint32) (hr error) = cfgmgr32.CM_Locate_DevNodeW
```

Comment:
Unless we need to dereference `pdnDevInst`, usual practice is to type it as a `uintptr`, so it's clear that it's an opaque pointer.